libs/core/models/register-credentials.ts

```diff
@@ -0,0 +1,18 @@
+/** Register credentials. */
+export type RegisterCredentials = {
+
+	/** Email. */
+	readonly email: string;
+
+	/** First name. */
+	readonly firstName: string;
+
+	/** Last name. */
+	readonly lastName: string;
+
+	/** Password. */
+	readonly password: string;
+
+	/** Avatar image url. */
+	readonly avatarUrl?: string;
```

Comment:
```suggestion
	readonly avatarUrl: string | null;
```
It is preferable to work with explicit values